package.json

```diff
@@ -58,25 +63,29 @@
     "sucrase": "^3.35.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.24.4",
-    "@babel/eslint-parser": "^7.25.7",
-    "@babel/plugin-syntax-import-assertions": "^7.25.7",
-    "@eslint/js": "^9.11.1",
-    "@ls-lint/ls-lint": "^1.10.0",
+    "@babel/core": "^7.28.5",
+    "@babel/eslint-parser": "^7.28.5",
```

Comment:
I updated babel because we're using it as our eslint parser, but I'm not sure we actually need it anymore since we're targeting ecma latest now.

Reply:
Also, totally unrelated, I just realized I called the TS Lint plugin **tseslint**, probably meant **tslint** 😅